deploy/crds/aws.managed.openshift.io_accountclaims_crd.yaml

```diff
@@ -133,16 +145,18 @@ spec:
                     description: Type is the type of the condition.
                     type: string
                 required:
-                - type
                 - status
+                - type
                 type: object
               type: array
             state:
+              description: ClaimStatus is a valid value from AccountClaim.Status
               type: string
           required:
           - conditions
           - state
           type: object
+      type: object
```

Comment:
[They tell me](https://github.com/openshift/gcp-project-operator/pull/113/files/cf2765203463d71dcea19e35a26156c6620f9c70#r515126411) this might cause problems on v3.11 (old hives). Do you have a way to test that?

Reply:
Hrm, that is unfortunate.  I had a minishift instance I could test this with until about a week ago. I will fire another one up and give it a try.

Reply:
See https://github.com/openshift/deadmanssnitch-operator/pull/88 for an example of how dms handled it, and how I changed it. (This all happened today.)

Reply:
Thx. Added a fixup make target with your `yq` cleanup example as a workaround.  I'll do this through the upgrades to v0.16.0, then jump on adding the boilerplate stuff.

Tested with the "fixup"-ed CRDs with Minishift (3.11.x) with no issues.